src/components/CompareResults/SubtestsResults/SubtestsResultsTable.tsx

```diff
@@ -11,23 +11,21 @@ import SubtestsTableContent from './SubtestsTableContent';
 
 type SubtestsResults = {
   key: string;
+  // By construction, there should be only one item in the array. But if more
+  // than one subtests share the same name, then there will be more than one item.
+  // Can this happen? We're not sure.
```

Comment:
I considered changing the value to not be an array, but then decided that I didn't want to risk a regression.